src/js/views/patients/shared/components/date-filter/date-filter_views.js

```diff
@@ -188,7 +188,7 @@ const LayoutView = View.extend({
 
 const DateRanges = Picklist.extend({
   className: 'date-filter__ranges',
-  itemTemplate: hbs`{{formatMessage (intlGet "patients.shared.components.dateFilterComponent.relativeDate") relativeTo=id}}{{#if isSelected}}{{fas "check" classes="u-float--right"}}{{/if}}`,
+  itemTemplate: hbs`{{formatMessage (intlGet "patients.shared.components.dateFilterComponent.relativeDate") relativeTo=id}}{{#if isSelected}}{{fas "check"}}{{/if}}`,
```

Comment:
I had a hard time switching this over to use `isCheckable`. Couldn’t get the custom `itemTemplate` for this picklist to work, as the picklist component needs to handle that directly for the `isCheckable` checkmark to be shown properly.

Was getting this as a result:

![Screen Shot 2022-10-17 at 11 38 25 PM](https://user-images.githubusercontent.com/35355575/196336801-245406d6-0326-4636-a61d-713c5cc3ab76.png)

So I don’t think the collection data being passed to it is in the correct format? It’s seeing each picklist item and which one is selected. But the correct text isn’t being shown.

I created a quick/temporary solution with the CSS I added for the `.icon.fa-check` icon if we need it for the release. I figured we could just remove that once we figure out `isCheckable` for this.